src/west/app/main.py

```diff
@@ -479,6 +500,20 @@ def run_command(self, argv, early_args):
         # If we're running an extension, instantiate it from its
         # spec and re-parse arguments before running.
 
+        if not early_args.help and early_args.command_name != "help":
+            # Recursively replace alias command(s) if set
+            aliases = self.aliases.copy()
+            while early_args.command_name in aliases:
+                new_args = shlex.split(aliases[early_args.command_name].args)
+                # Make sure we don't end up in an infinite loop
+                del aliases[early_args.command_name]
+                self.queued_io.append(lambda cmd: cmd.dbg(
+                    f'Replacing alias {early_args.command_name} with {new_args}'
```

Comment:
While trying to see this log with `west -v mydiff` I'm afraid I found a command parsing issue

```shell
west config alias.mydiff "diff --stat"

west      diff  # ok
west -v   diff  # ok
west    mydiff  # ok
west -v mydiff  # fails

usage: west [-h] [-z ZEPHYR_BASE] [-v] [-V] <command> ...
west: unknown command "mydiff"; workspace ... does not define this extension command ...
```